plugins/modules/portage.py

```diff
@@ -121,6 +121,12 @@
     type: bool
     default: false
 
+  select:
+    description:
+      - Explicitely add the package to the world file
+    type: bool
+    default: false
```

Comment:
```suggestion
    default: false
    version_added: 8.6.0
```

Reply:
Actually, does `default: false` is needed? This ensures that `--select n` is always passed when the option is not explicitly specified by the user. Could this have any unwanted side-effects?

Reply:
You're right! I missed that when I reworked my proposal (originally I was thinking to have `select:false` not adding the flag at all but I changed my mind to be more aligned with the rest of the code!)

I'm updating my PR and will re-test